packages/shared/components/layouts/website-section/feed.marko

```diff
@@ -6,6 +6,11 @@ $ const { config, GAM } = out.global;
 $ const { id, alias, name, pageNode } = input;
 $ const description = defaultDescription(name, config);
 
+$ const image = (input.image) ? input.image : {};
```

Comment:
Replace this with `input.listFlow` and spread to both the flow via `<shared-content-list-flow ...input.listFlow nodes=nodes>` and load more via`<@component-input ...input.listFlow />`